main.go

```diff
@@ -1,5 +1,67 @@
 package main
 
+import (
+	"context"
+	"log"
+
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/config"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/db"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/forman"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/reconciler"
+	"github.com/red-hat-storage/managed-fusion-fleet-reconciler/pkg/types"
+
+	"go.uber.org/zap"
+)
 
 func main() {
+	// parse configuration from yaml file
+	conf, err := config.GetConfig("config/config.yaml")
```

Comment:
i dont see the reason to make it configurable as we decide how to deploy it. If you feel it needs to be changed, please let me know. sounds good to add it do dockerignore not gitignore

Reply:
> make it configurable
- if we can always ensure this is mounted relative to exe I don't have any issue

> add it do dockerignore not gitignore
- if devs tries to update the config to prod creds I thought it's better to have this file ignored rather than committed by mistake

Reply:
Moving this to be provided by the environment makes sense. The repo itself has no notion of deployment and as such of deployment location. I will vote to have this provided by the environment via and env var (CONFIG_FILE)